apps/web/components/booking/BookingListItem.tsx

```diff
@@ -61,9 +61,9 @@ function BookingListItem(booking: BookingItemProps) {
     };
     /**
      * Only pass down the recurring event id when we need to confirm the entire series, which happens in
-     * the "Recurring" tab, to support confirming discretionally in the "Recurring" tab.
+     * the "Recurring" tab and "Unconfirmed" tab, to support confirming discretionally in the "Recurring" tab.
      */
-    if (booking.listingStatus === "recurring" && booking.recurringEventId !== null) {
+    if ((isTabRecurring || isTabUnconfirmed) && isRecurring) {
```

Comment:
Are these constants being used before defined? We're at line 66 and the constants are defined in line 79-80

Reply:
The usage in that line comes from a function used inside an arrow function after the variables are defined. Shouldn't be a problem.

Reply:
I'm aware, but even so the function is being defined before them and it depends on variables in the upper scope that are defined after. If this was vanilla JS this would be an error. Also it's confusing.

Reply:
I can move the function to be below the constants definition, that would help I guess, would that work?